src/common/constants.js

```diff
@@ -1,7 +1,11 @@
-export const API_KEY = 'f0d4a67a9a6d56c02175828269721e4e';
-export const BASE_URL = 'http://api.openweathermap.org/data/2.5/weather';
+export const BASE_URL = 'https://pfa.foreca.com/api/v1';
```

Comment:
Do you really need it on the client side?

Reply:
Yes, we're making the requests on the client